modules/ui/src/main/scala/lucuma/ui/components/state/ConnectionManager.scala

```diff
@@ -42,7 +42,8 @@ object ConnectionManager {
         import props.given
 
         (Logger[DefaultA].debug(s"[ConnectionManager] Token changed. Refreshing connections.") >>
-          props.openConnections(props.payload)).whenA(initializedState.value)
+          props.closeConnections >> props.openConnections(props.payload)).uncancelable
+          .whenA(initializedState.value)
```

Comment:
Does this apply to the whole action or just the openConnections?

Reply:
You mean the `uncancelable`? It applies to the whole thing

Reply:
Actually I think it doesn't:

<img width="534" alt="afbeelding" src="https://github.com/gemini-hlsw/lucuma-ui/assets/10114577/bea3513d-41dd-4cea-a4b1-f3881806fdc2">
<img width="208" alt="afbeelding" src="https://github.com/gemini-hlsw/lucuma-ui/assets/10114577/379ad1ea-e88a-4ac1-95b5-becf433cf902">

Reply:
I believe there are parenthesis. The opening one is prior to `Logger`. The closing is right before `.uncancelable`

Reply:
Oop yeah I missed that. Should be fine then

Reply:
Right. I'm pushing a commit reformatting it so it's clearer.